File: src/main/java/com/example/kakebo/entity/LogDiario.java
package com.example.kakebo.entity;



import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;

@Entity
public class LogDiario {
  
  @Id
  @GeneratedValue(strategy = GenerationType.AUTO)
  @Column(name = "id")
  @JsonProperty("id")
  private Long id;
  
  @Column(name="fecha")
  @JsonProperty("fecha")
  private Date fecha;

  @Column(name="monto")
  @JsonProperty("monto")
  private double monto;

  @ManyToOne
  @JoinColumn(name ="idCategoria", referencedColumnName = "id")
  @JsonIgnore
  private Categoria categoria;


  public Date getFecha() {
    return fecha;
  }

  public void setFecha(Date fecha) {
    this.fecha = fecha;
  }

  public double getMonto() {
    return monto;
  }

  public void setMonto(double monto) {
    this.monto = monto;
  }

  public Categoria getCategoria() {
    return categoria;
  }

  public void setCategoria(Categoria categoria) {
    this.categoria = categoria;
  }
}

File: src/main/kotlin/org/example/kakebo/entity/LogDiario.kt
package org.example.kakebo.entity

import io.quarkus.hibernate.orm.panache.PanacheEntity
import java.util.*
import javax.persistence.*

@Entity
class LogDiario: PanacheEntity() {

  var fecha:Date?=null
  var monto:Double=0.0
  @Column( insertable = false, updatable = false)
  var idCategoria:Long=0

  @ManyToOne(fetch = FetchType.EAGER)
  @JoinColumn(name = "idCategoria")
  var cat:Categoria?=null
}